test/app/features/escape_manual/data/model/escape_manual_fixtures.dart

```diff
@@ -12,19 +24,39 @@ final escapeManualModelFixture = EscapeManualRemoteModel(
   ),
   tasks: [
     EscapeManualTaskRemoteModel(
-      id: 'id',
-      type: 'type',
+      id: '1',
+      type: 'checkbox',
       group: 'group',
       title: 'title',
       description: 'description',
-      isEditable: true,
+      isEditable: false,
+      isDone: false,
       userInputValue: 'userInputValue',
```

Comment:
Faz sentido esse valor num `checkbox` não editável?

Reply:
Sinceramente eu não sei se faria sentido, pois o `isEditable` foi mantido apenas para ficar semelhante ao retorno da API e não é utilizado para mais nada no aplicativo, talvez o melhor seria remover essa propriedade?

Reply:
Idem aqui, estou me referindo ao `userInputValue`

Reply:
Coloquei um valor aleatório somente para testar a lógica, pois ainda não tem a funcionalidade que atribui um valor real/válido para essa propriedade, de qualquer maneira mudei para `null` e quando a funcionalidade for integrada deixo o tipo que faça mais sentido.